README.md

```diff
@@ -1,37 +1,178 @@
 # Description
 
-Python libraries interfacing different mpi implementations.
+`mympirun` is a tool to make it easier for users of HPC clusters to run MPI programs with good performance.
+
+It wraps around the `mpiexec`/`mpirun`/... commands that are provided by the different MPI implementations (OpenMPI, Intel MPI, ...).
+
+`mympirun` will determine the `mpirun` command to use, sets up the environment, and takes into account the available resources to configure it.
 
 Originally created by the [HPC team of Ghent University](http://ugent.be/hpc).
 
+
 # License
 
-`vsc-mympirun` is made available under the GNU General Public License
-(GPL) version 2.
+`vsc-mympirun` is made available under the GNU General Public License (GPL) version 2.
 
 
 # Workflow
```

Comment:
Rename this to `Internal workflow`, and move it (way) down, most users shouldn't care about this.

The `Basic usage` section should go here.